display/pfobjects.py

```diff
@@ -116,8 +123,12 @@ def draw(self, projection, opt=''):
             
 class GStraightTrajectory(GTrajectory):
     def __init__(self, description):
-        super(GStraightTrajectory, self).__init__(description,
-                                                  linestyle=2, linecolor=1)
+        self.desc = description
+        if abs(self.desc.pdgid()) in [22]:
+          super(GStraightTrajectory, self).__init__(description,
+                                                  linestyle=2, linecolor=2)
+        elif abs(self.desc.pdgid()) in [12, 14, 16]:
```

Comment:
Actually heppy now displays reconstructed particles. Since neutrinos are not reconstructed, they won't be displayed. 

We could think about displaying generated particles, in which case your change can be useful. But you will need to do this to test this change as currently, you don't have neutrinos. So please revert this part to the previous version. 

Why do you need a display of neutrinos?